bot/cogs/ext/commands/anime.py

```diff
@@ -191,6 +191,9 @@ async def cancer_owo(self, ctx: Context, *, text: str = None):
     @commands.max_concurrency(number=1, per=commands.BucketType.user)
     async def waifu(self, ctx: Context, *, selection: str = None):
         async with ctx.typing():
```

Comment:
Should we still use the typing event?

Reply:
no idea, plugin command uses it so i left it in

Reply:
when a cached result is found, it is finished fast enough that typing event isnt needed, but if cache misses it takes a second

Reply:
That's the problem, if the result is delivered fast, the typing will continue for 5 seconds which is bad UX, imho

Reply:
check the latest commit, although i think youre going to want some changes to it